client_code/navigation.py

```diff
@@ -78,7 +78,7 @@ def _default_link_click(**event_args):
     """
     _reset_links()
     link = event_args["sender"]
-    link.role = ["selected"]
+    link.role.append("selected")
```

Comment:
This won't trigger a change to the dom, you'll need to assign back to the role property